src/notify.js

```diff
@@ -28,6 +28,8 @@ function Notify(title, options = {}) {
         notifyError = null,
         closeOnClick = false,
         timeout = null,
+        swUrl = null,
```

Comment:
Also, could you potentially do what you need here without this? Seems like you could pass a message to the service worker and fire a notification, instead of use it this way. Just an idea.